ha/setup/ha_setup.py

```diff
@@ -633,7 +634,37 @@ def process(self):
         """
         Process reset command.
         """
-        pass # TBD
+        Log.info("Processing reset command")
+        try:
+            # create new version of log file
+            services = ["pcsd", "corosync", "cortx_ha_log.conf", "pacemaker"]
+            for service in services:
+                self._execute.run_cmd(f"logrotate --force /etc/logrotate.d/{service}")
+
+            older_logs = []
+            log_dirs = [const.COROSYNC_LOG, const.RA_LOG_DIR, "/var/log/pcsd"]
+            for log_dir in log_dirs:
+                log_list = self._execute.run_cmd(f"ls -I *.log -I *.xml {log_dir}")
```

Comment:
why is command being used here. For listing the file use os module of python and files can be found out from there.

Reply:
Used python os module here